### DISCUSSION TITLE
How to Filter scenes by file size

### DISCUSSION THREAD

**Discussion by misterscv** [stashapp/stash#4898] [Q&A]
Hi. Help me with Filter by file size. For example "more than 1000mb" or something like this. Stash knows about file size of the scenes. I can see this info in "file Info > file size" But can not filter scenes by this parameter.  Thanks

**Comment by DogmaDragon** [comment]
Not possible at the moment. You can make a feature request https://github.com/stashapp/stash/issues/new/choose.
